llama-stack-client-kotlin-client-local/src/main/kotlin/com/llama/llamastack/client/local/LlamaStackClientClientLocalImpl.kt

```diff
@@ -0,0 +1,98 @@
+// File generated from our OpenAPI spec by Stainless.
+
+package com.llama.llamastack.client.local
+
+import com.llama.llamastack.client.LlamaStackClientClient
+import com.llama.llamastack.client.LlamaStackClientClientAsync
+import com.llama.llamastack.models.*
+import com.llama.llamastack.services.blocking.*
+
+class LlamaStackClientClientLocalImpl
```

Comment:
What's the story with this class? Can we just kill it?

Reply:
We want to use the same structure/parent classes as the remote (http) client side. An example is that with remote inferencing we would call by: `client!!.inference().chatCompletion(...)`

With keeping this, we can also similarly do local inference calling like `client!!.inference().chatCompletion(...)`.

trying to keep things as close as possible for local vs remote to maintain ease for developer.